app/main/views/errors.py

```diff
@@ -1,9 +1,9 @@
 from flask import render_template
```

Comment:
Why was this moved into `views`? It makes the layout different to the other frontend apps and they're a bit distinct from the other modules in the `views` package.